### DISCUSSION TITLE
Missing providerconfig CRD

### DISCUSSION THREAD

**Discussion by astropanic** [crossplane/crossplane#5865] [General]
I’m new to Crossplane and I’m following the [tutorial](https://docs.crossplane.io/latest/getting-started/provider-aws/).

> Create a ProviderConfig
A ProviderConfig customizes the settings of the AWS Provider.
Apply the ProviderConfig with the this Kubernetes configuration file:

The problem is I don’t have a CRD `ProviderConfig` on my cluster.

```
k api-resources | grep cross | grep -i provider
```

```
providerrevisions                                         pkg.crossplane.io/v1                        false        ProviderRevision
providers                                                 pkg.crossplane.io/v1                        false        Provider
```

How I’m supposed to create a ProviderConfig when my cluster doesn’t understand what it is?

**Comment by jbw976** [comment]
Did you already install the AWS provider as well, as per https://docs.crossplane.io/latest/getting-started/provider-aws/#install-the-aws-provider? Each provider brings their own `ProviderConfig` object as part of their installation, so perhaps something is wrong with the provider's installation.

What is the output of `kubectl get pkg` and `kubectl get providerrevisions`?
